packages/happn-cluster/lib/services/orchestrator/constants.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
look at moving this into a constants area, ie: packages/happn-cluster/lib/constants/orchestrator-states.js

Reply:
Done